Lib/enum.py

```diff
@@ -427,8 +431,8 @@ def __new__(metacls, cls, bases, classdict, *, boundary=None, _simple=False, **k
         # check for illegal enum names (any others?)
         invalid_names = set(member_names) & {'mro', ''}
         if invalid_names:
-            raise ValueError('Invalid enum member name: {0}'.format(
-                ','.join(invalid_names)))
+            raise ValueError('invalid enum member name(s) '.format(
```

Comment:
Is removing "{0}" proper?

Reply:
@gryznar Only because it was changed to `%s` later.